ui/components/app/gas-customization/gas-price-button-group/index.scss

```diff
@@ -195,7 +195,7 @@
     color: var(--color-text-default);
     width: 108px;
     height: 97px;
-    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.151579);
+    box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
     border-radius: 6px;
     border: none;
 
```

Comment:
Don't think this component is used. Made a quick story to display that isn't in this commit

<img width="1440" alt="Screen Shot 2022-07-15 at 6 00 36 PM" src="https://user-images.githubusercontent.com/8112138/179328177-0b58730d-66e0-4537-aa0b-272f4b572c57.png">